create-site.sh

```diff
@@ -131,47 +76,45 @@ letsencrypt certonly --standalone -d $DOMAIN
 service php7.0-fpm restart
 service nginx start
 
-# create random password
-PASSWDDB="$(openssl rand -base64 12)"
-
-# replace "-" with "_" for database username
-MAINDB=${USER_NAME//[^a-zA-Z0-9]/_}
-
-# If /root/.my.cnf exists then it won't ask for root password
-if [ -f /root/.my.cnf ]; then
-    mysql -e "CREATE DATABASE ${USER} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
-    mysql -e "CREATE USER ${USER}@localhost IDENTIFIED BY '${PASSWDDB}';"
-    mysql -e "GRANT ALL PRIVILEGES ON ${USER}.* TO '${USER}'@'localhost';"
-    mysql -e "FLUSH PRIVILEGES;"
-# If /root/.my.cnf doesn't exist then it'll ask for root password
-else
-    echo -n "Please enter root user MySQL password!"
-    read -s rootpasswd
-    mysql -uroot -p${rootpasswd} -e "CREATE DATABASE ${USER} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
-    mysql -uroot -p${rootpasswd} -e "CREATE USER ${USER}@localhost IDENTIFIED BY '${PASSWDDB}';"
-    mysql -uroot -p${rootpasswd} -e "GRANT ALL PRIVILEGES ON ${USER}.* TO '${USER}'@'localhost';"
-    mysql -uroot -p${rootpasswd} -e "FLUSH PRIVILEGES;"
-fi
-
 read -p "Install WP (Y/n)? " -n 1 -r
 echo ""
 
 if [[ $REPLY =~ ^[Yy]$ ]]; then
+    PASSWDDB="$(openssl rand -base64 12)"
+
+    # replace "-" with "_" for database username
+    MAINDB=${USER_NAME//[^a-zA-Z0-9]/_}
+    # If /root/.my.cnf exists then it won't ask for root password
+    if [ -f /root/.my.cnf ]; then
+        mysql -e "CREATE DATABASE ${USER} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
+        mysql -e "CREATE USER ${USER}@localhost IDENTIFIED BY '${PASSWDDB}';"
+        mysql -e "GRANT ALL PRIVILEGES ON ${USER}.* TO '${USER}'@'localhost';"
+        mysql -e "FLUSH PRIVILEGES;"
+    # If /root/.my.cnf doesn't exist then it'll ask for root password
+    else
+        echo -n "MySql Root Password: "
+        read -s rootpasswd
+        mysql -uroot -p${rootpasswd} -e "CREATE DATABASE ${USER} /*\!40100 DEFAULT CHARACTER SET utf8 */;"
+        mysql -uroot -p${rootpasswd} -e "CREATE USER ${USER}@localhost IDENTIFIED BY '${PASSWDDB}';"
+        mysql -uroot -p${rootpasswd} -e "GRANT ALL PRIVILEGES ON ${USER}.* TO '${USER}'@'localhost';"
+        mysql -uroot -p${rootpasswd} -e "FLUSH PRIVILEGES;"
+    fi
     sudo -u $USER -i -- wp core download --path=htdocs
     sudo -u $USER -i -- wp core config --path=htdocs --dbname=$USER --dbuser=$USER --dbpass=${PASSWDDB}
 
+    if [[ -z $WP_ADMIN ]]; then
```

Comment:
I'd rather have these data under a setup flag. If that flag isn't specified, then don't ask for all of these.

Something like `create-site.sh --interactive`